recipes/django-modelcluster/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "django-modelcluster" %}
+{% set version = "3.1" %}
+{% set sha256 = "9edde7fb87b413c1114df47f928e95fbd6878de17f9f2caeea90e77766e272b5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
How about `noarch: python` here?

Reply:
Thanks. I will add that to the build section

Reply:
Why is it running for all OS? It should run only cirlceci. I think I need to rerender the feedstock for noarch to be effective

Reply:
It does needs a rerender, but staged-recipes only does that when you close/open the PR I think.

Reply:
`staged-recipes` will run in all CIs, but the feedstock will run only in CircleCI and doesn't require a rerender.

Reply:
https://github.com/conda-forge/tox-feedstock/pull/7/files
For tox, it runs only circleci, but for dajngo-modelcluster it is running for all OS.

Reply:
@sannykr That's because it's on `tox-feedstock`. If you look back at the staged-recipes PR: https://github.com/conda-forge/staged-recipes/pull/3700, it was actually running on all CIs.

Reply:
@dougalsutherland: got it. Thanks